axlearn/open_api/metrics/tool_use_execution.py

```diff
@@ -342,4 +514,9 @@ def get_tool_calls_from_message(message: dict[str, Any]) -> list[dict[str, Any]]
         "number_of_examples": len(responses),
         "number_of_parsing_errors": number_of_parsing_errors,
         "number_of_generation_errors": number_of_generation_errors,
+        "func_name_accuracy": _safe_div(total_func_name_matches, total_tool_calls),
```

Comment:
We can probably just do
```suggestion
        "func_name_accuracy": total_func_name_matches / max(1, total_tool_calls),
```

Reply:
Done